File: web/app/model/wip/SrOutsourceDetl.js
Ext.define('iFlat.model.wip.SrOutsourceDetl', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'srOutsourceDetl.id', mapping: 'id', type: 'string'},
        {name: 'srOutsourceDetl.pid', mapping: 'pid', type: 'string'},
        {name: 'srOutsourceDetl.content', mapping: 'content', type: 'string'},
        {name: 'srOutsourceDetl.specs', mapping: 'specs', type: 'string'},
        {name: 'srOutsourceDetl.unit', mapping: 'unit', type: 'string'},
        {name: 'srOutsourceDetl.qty', mapping: 'qty', type: 'number'},
        {name: 'srOutsourceDetl.comment', mapping: 'comment', type: 'string'},
    ]
});

File: src/main/java/com/iflat/wip/bean/SrOutsourceDetl.java
package com.iflat.wip.bean;

/**
 * Created by tyriv on 2016/9/1.
 */
public class SrOutsourceDetl {

    private String id;
    private String pid;
    private String content;
    private String specs;
    private String unit;
    private double qty;
    private String comment;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPid() {
        return pid;
    }

    public void setPid(String pid) {
        this.pid = pid;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getSpecs() {
        return specs;
    }

    public void setSpecs(String specs) {
        this.specs = specs;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public double getQty() {
        return qty;
    }

    public void setQty(double qty) {
        this.qty = qty;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }
}
